ansible/playbook/openstack_vm_remove_aggregate.yml

```diff
@@ -0,0 +1,15 @@
+---
+# Variables:
+#  - "vm_name": "n311-test"
+- name: "Delete Server on Openstack"
+  import_playbook: "openstack_vm_delete.yml"
+  vars:
+    hetzner_context_name: "snowdrop"
```

Comment:
Same remark about the var as mentioned before !

Reply:
the hetzner variable definition has been removed as it wasn't used.